jaxopt/_src/lbfgs.py

```diff
@@ -211,6 +213,7 @@ class LBFGS(base.IterativeSolver):
 
   stepsize: Union[float, Callable] = 0.0
   linesearch: str = "zoom"
+  allow_failed_linesearch: bool = False
```

Comment:
So the default behavior now is the same as that of LBFGS in JAX core, right?  (just to confirm)

Reply:
I'm confused. I thought `allow_failed_linesearch == True` was JAXopt's behavior? (since the optimization loop keeps going even in the case of failed line search).

Reply:
Ah it's because there is a confusion about the naming. Originally I had called it `allow_linesearch_failure` which meant that we authorize the linesearch failure to be passed on. Now with the new name I understand why there is some confusion. Maybe we can just change the name?

Reply:
My opinion is that I would keep `stop_if_linesearch_fails=False` by default for the time being because we already have users relying on LBFGS as (inner) solver and we can't take the risk to break their code/tests without any warning. If there is strong evidence that stopping is the right thing to do and this should be the default, we need to use a deprecation cycle to warn that the default value is going to change in release 0.x

Reply:
Ok for now it is `stop_if_linesearch_fails=False`, keeping it as is.